architecture/utilisation-concept-canadaca.md

```diff
@@ -31,7 +31,7 @@ title: "Qui doit utiliser le sysème de conception de Canada.ca"
       <li>les entreprises ou les  organisations qui ne font pas partie du gouvernement du Canada</li>
     </ul>
     <h2>Utiliser la Boîte à outils de l'expérience Web</h2>
-    <p>On encourage toutes les institutions à utiliser les composants de la&nbsp;<a href="/fr/secretariat-conseil-tresor/services/communications-gouvernementales/boite-outils-experience-web.html">Boîte à outils de l'expérience Web</a>&nbsp;pour leurs produits internes et externes. Ces composants sont gratuits et disponibles de façon ouverte. Ils reflètent les résultats des tests de convivialité.  Ils facilitent la  conformité aux exigences en matière d'accessibilité et de conception réactive  pour une diversité d'appareils avec un large éventail de tailles d'écran.</p>
+    <p>On encourage toutes les institutions à utiliser les composants de la&nbsp;<a href="https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/boite-outils-experience-web.html">Boîte à outils de l'expérience Web</a>&nbsp;pour leurs produits internes et externes. Ces composants sont gratuits et disponibles de façon ouverte. Ils reflètent les résultats des tests de convivialité.  Ils facilitent la  conformité aux exigences en matière d'accessibilité et de conception réactive  pour une diversité d'appareils avec un large éventail de tailles d'écran.</p>
```

Comment:
In English the link points to http://www.tbs-sct.gc.ca/hgw-cgf/oversight-surveillance/communications/ws-nw/wet-boew-eng.asp. The one you've linked, is this a new page?

Reply:
I'm not sure which link is correct. No links were provided to me. I just fixed the broken link paths with paths that would work.

Reply:
@donohuech do you know which link it should be, please?